gatling-core/src/main/scala/io/gatling/core/ConfigKeys.scala

```diff
@@ -158,7 +158,6 @@ object ConfigKeys {
       val ApiKey = "gatling.data.datadog.apiKey"
       val AppKey = "gatling.data.datadog.appKey"
       val Site = "gatling.data.datadog.site"
-      val BufferSize = "gatling.data.datadog.bufferSize"
```

Comment:
Was this added just in the fork? If so no problem removing it

Reply:
Yes I added this in my previous PR when I was sending the raw data to datadog. As the new setup just sends the aggregations we don't need to worry about buffer size.